pyproject.toml

```diff
@@ -8,3 +8,9 @@ requires = ["setuptools",
             "Cython>=0.29.21,<3.0"]
 
 build-backend = 'setuptools.build_meta'
+
+[tool.cibuildwheel]
+# Skip pypy on mac due to numpy/accelerate issues
+skip = "pp37-macosx_x86_64"
```

Comment:
I think we could skip pypy for all platforms. I'm not sure how binary wheels are supported on pypy, and astropy doesn't build pypy wheels.